client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -99,6 +101,10 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 		? reduxHideFreePlan && 'plans-blog-onboarding' === plansIntent
 		: reduxHideFreePlan;
 
+	const onPlanTypeSelectorChange = ( path: string ) => {
+		navigate( path );
```

Comment:
**Note:**

Stepper uses `react-router` for client side routing, so it needs its own specific logic to handle plan type selector selection